api_docs/changelog.md

```diff
@@ -674,7 +679,7 @@ No changes; feature level used for Zulip 5.0 release.
 
 **Feature level 83**
 
-* * [`POST /register`](/api/register-queue): The `cross_realm_bots`
+  * * [`POST /register`](/api/register-queue): The `cross_realm_bots`
```

Comment:
This change looks like a typo.

Reply:
Right, it seems like it was my formatter messing with markdown files, fixed it!